xscen/aggregate.py

```diff
@@ -0,0 +1,455 @@
+import datetime
+import logging
+from pathlib import Path
+from typing import Union
+
+import geopandas as gpd
+import numpy as np
+import pandas as pd
+import xarray as xr
+import xesmf as xe
+from shapely.geometry import Polygon
+
+from .config import parse_config
+from .extract import clisops_subset
+
+logger = logging.getLogger(__name__)
+
+__all__ = ["climatological_mean", "compute_deltas", "spatial_mean"]
+
+
+@parse_config
+def climatological_mean(
+    ds: xr.Dataset,
+    *,
+    window: int = None,
+    min_periods: int = None,
+    interval: int = 1,
+    periods: list = None,
+    to_level: str = None,
```

Comment:
```suggestion
    to_level: str = 'climatology',
```
Je mettrais un défaut juste pour que la catalogue n'ait pas une entrée vide si la personne ne spécifie pas de level.